README.md

```diff
@@ -10,75 +10,80 @@
-scalable IP networking principles as the internet: providing connectivity using
-a pure Layer 3 approach.  Calico can be deployed without encapsulation or
-overlays to provide high performance at massive scales.
-
-Read more about it on the [Project Calico website](https://www.projectcalico.org).
-
-When using Calico networking in containerized environments, each container
-gets its own IP and fine grain security policy.  A `calico-node` service runs
-on each node which handles all of the necessary IP routing, installation of
-policy rules, and distribution of routes across the cluster of nodes.
-
-This repository contains:
--  The `calico/node` container Dockerfile and build environment.  It contains
-  the configuration and "glue" that pull together four separate processes to
-  provide Calico networking:
-  * Felix, the Calico worker process
-  * BIRD, the route distribution process
-    (there are separate processes for IPv4 and IPv6)
-  * Confd, a templating process to auto-generate configuration for BIRD
--  A command line tool, `calicoctl`, which makes it easy to configure
-   and start the Calico service listed above, and allows you to interact with
-   the datastore (etcd) to define and apply rich security policy to the
-   containers you create.
-
-Please refer to our [main documentation](http://docs.projectcalico.org) for details on deploying Calico and 
-using `calicoctl`.
-
-## Common set-up
-
-Assuming you have already installed **go version 1.7.1+**, perform the following simple steps to get building:
-
-- [Install Glide](https://github.com/Masterminds/glide#install)
-
-- Clone this repository to your Go project path: 
-```
-git clone git@github.com:projectcalico/calico-containers.git $GOPATH/src/github.com/projectcalico/calico-containers
-```
+This repository is the home of `calico/node` and `calicoctl`.
+
+#### For information on how to get started using Calico see: https://docs.projectcalico.org
+
+- For information on `calico/node`, see [documentation on calico/node architecture](http://docs.projectcalico.org/master/reference/architecture/components).
+
+- For information on `calicoctl` usage, see [calicoctl reference information](http://docs.projectcalico.org/master/reference/calicoctl/)
+
+### Developing
+
+Calico-containers is a golang project, so assuming you have already installed **go version 1.7.1+**, clone this repository into your Go project path:
 
-- Switch to your project directory:
 ```
-cd $GOPATH/src/github.com/projectcalico/calico-containers
+git clone https://github.com/projectcalico/calico-containers.git $GOPATH/src/github.com/projectcalico/calico-containers
 ```
 
-- Populate the `vendor/` directory in the project's root with this project's dependencies:
+Useful actions can be printed by running `make help` in the repo root.
+
+### Building `calico/node`
+
+To build the `calico/node` container, run the `make calico/node` build step from
```

Comment:
This would be more clear in code blocks. e.g

To build the `calico/node` container, run the following make target from the root of the repository:

```
make calico/node
```